### DISCUSSION TITLE
Change to landscape mode on reaching Question set in an Interactive video

### DISCUSSION THREAD

**Discussion by Xitija** [Sunbird-Ed/Community#15] [Enhancement request]
Hello @swayangjit Sir,

I am trying to run an Interactive Video in the mobile app .
When the video reaches the Question Set the UI is breaking .

![image](https://user-images.githubusercontent.com/65657373/170699926-473a8e84-1abd-4149-ab12-3e8b6f10855b.png)
We want to open the Question Set in landscape mode ,
Can you please tell what code is to be added / updated for doing that ?
This is app is built on local setup after addig the video player package , the UI breaking issue persists in the existing app , also it is pointing to https://staging.sunbirded.org/ .

cc: @Krishnaj20 @coolbung @baljeetsingh

**Comment by swayangjit** [comment]
@Xitija You can take a pull from the "release-4.9.0" branch and verify the changes or you can manually change the code by refer following PR https://github.com/Sunbird-Ed/SunbirdEd-mobile-app/pull/3016

**Reply by Xitija** [reply]
Interactive video opens in landscape mode and can be played in landscape mode only. For now I am marking this as answer I have tested this by manually making code changes on 4.8.1 .
Tested 0n 4.9 along with video player package. Working here as well
